LEAF_Request_Portal/db_mysql.php

```diff
@@ -1,347 +0,0 @@
-<?php
```

Comment:
Not sure about this either.

Reply:
This one should be deleted. When I was going through every file, and noticing an issue with what db file was being used and then noticing there really wasn't much of a difference between the two files (Nexus and Portal) I decided to combine them. That made the two obsolete so yes I will delete this one.